torch/jit/_serialization.py

```diff
@@ -77,8 +78,13 @@ def forward(self, x):
     """
     if _extra_files is None:
         _extra_files = {}
-    if isinstance(f, str) or isinstance(f, pathlib.Path):
-        m.save(f, _extra_files=_extra_files)
+    if isinstance(f, (str, pathlib.Path)):
```

Comment:
Thanks for making the changes. Newly added code seems a bit low-level though, do you mind adding them to somewhere here?

https://github.com/pytorch/pytorch/blob/5654fc8edd56c8640d88fa325417ed406d3e2f73/torch/csrc/jit/serialization/export_module.cpp#L314

Reply:
Sure, I can add test to python-side with no problems but it would take me way too long to move the code to cpp side as I don't have that much experience with it.